haskell-src/Concordium/ID/Types.hs

```diff
@@ -949,6 +949,12 @@ data AccountCredential' credTy
     | NormalAC (CredentialDeploymentValues' credTy) CredentialDeploymentCommitments
     deriving (Eq, Show, Functor)
 
+credIdFromRaw :: RawCredentialRegistrationID -> Either String CredentialRegistrationID
```

Comment:
Docs, including the precondition.

Reply:
I agree about the docs, but I don't think there is any precondition as such.